lib/crypto-tools/keyring.js

```diff
@@ -273,4 +281,95 @@ KeyRing.prototype._decrypt = function(data) {
   return JSON.parse(dec) && dec;
 };
 
+/**
+ * Retrieve the bucketKey
+ * @param {String} bucketId - bucket id
+ * @returns {String|null}
+ */
+KeyRing.prototype.generateBucketKey = function(bucketId) {
+  if(!this._mnemonic || !bucketId){
```

Comment:
Use spaces in between `if/else` and `()` and `{` to stay consistent with the styles and conventions elsewhere:

```
if (!this._mnemonic || !bucketId) {
```